src/ArraySorter.php

```diff
@@ -75,21 +75,21 @@ public static function multisort(array &$array, $key, $direction = SORT_ASC, $so
     /**
      * Get keys for get arguments
      *
-     * @param array $array the array to be sorted. The array will be modified after calling this method.
+     * @param array $array the array to be sorted
      * @param string|\Closure|array $key the keys to be sorted by. This refers to a key name of the sub-array
      * elements, a property name of the objects, or an anonymous function returning the values for comparison
      * purpose. The anonymous function signature should be: `function($item)`.
      * To sort by multiple keys, provide an array of keys here.
      * @return array return the keys
      */
-    private static function getKeys(array &$array, $key): array
+    private static function getKeys(array $array, $key): array
     {
         $keys = is_array($key) ? $key : [$key];
         if (empty($keys) || empty($array)) {
             return [];
         }
 
-        if (is_callable($key)) {
+        if ($key instanceof \Closure) {
```

Comment:
That looks suspicious. What is the reason for this change?

Reply:
ArrayHelper methods (in particular, `getColumn` that used in this method) don't support callable as array `$key = [$this, 'methodName']` or named function `$key = 'fnName'`. This is occurs error. 

Also in phpdoc for this function specified `string|\Closure|array`, not `callable`.